pkg/buildkit/buildkit.go

```diff
@@ -118,6 +119,25 @@ type Client struct {
 	dockerAuthConfig string
 }
 
+func validateCompression(compression hephconfig.BuildkitCompression, name string) map[string]string {
```

Comment:
we can leverage the compression string types we define in the cfg package here ->

```
func validateCompression(compression hephconfig.BuildkitCompression, name string) map[string]string {
	attrs := map[string]string{
		"name": name,
	}

	switch compression {
	case hephconfig.CompressionStargz:
		attrs["push"] = "true"
		attrs["compression"] = "estargz"
		attrs["force-compression"] = "true"
		attrs["oci-mediatypes"] = "true"
	case hephconfig.CompressionZstd:
		attrs["compression"] = "zstd"
		attrs["force-compression"] = "true"
	case hephconfig.CompressionGzip:
		attrs["push"] = "true"
	}

	return attrs
}
```

also, more of a nit/naming thing but it might be better to use something like `setCompressionAttrs` since we're not performing validation in any strict sense 

we could extend that example above more if we wanted some additional validation and have `default` return an error for unsupported compression types

Reply:
fixed in latest update